Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -358,18 +359,22 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             return 1
 
         commits = list(repository.commits(begin=dict(hash=branch_point.hash), end=dict(branch=repository.branch)))
-        issues = commits[0].issues
+        issues = list(chain(*[commit.issues for commit in commits]))
```

Comment:
I think this is better expressed as a list comprehension:
```suggestion
        issues = [
            issue 
            for commit in commits
            for issue in commit.issues
        ]
```